clink/app/src/loader/clink.bat

```diff
@@ -5,6 +5,19 @@
 set clink_profile_arg=
 set clink_quiet_arg=
 
+set clink_autorun=0
+set clink_parameters="%*"
+set clink_parameters_nospace="%clink_parameters: =_%"
+if not %clink_parameters_nospace:--autorun=%==x%clink_parameters_nospace% set clink_autorun=1
```

Comment:
I think an `x` went missing while addressing the case I noted.  Since only the right side of the `==` has an `x`, the `==` will always evaluate to false, so the full `not` condition will always be true, so it will always `set clink_autorun=1` and it will always bail out regardless whether the `--autorun` flag is present.

If that's the only issue that I find while testing this, then I'll accept the merge and fix it after the merge.